### ISSUE TITLE
Invoice Edit

### ISSUE THREAD

**Issue by meakajon** [SolidInvoice/SolidInvoice#303]
When editing a previously created invoice, the Qty value is set to 1 instead of the amount that was entered in the originally created invoice. For example I have added to an invoice a monthly charge, so for the year I have set the Qty to 12. But on edit of the invoice it sets this to 1 instead of 12.

1. Create New Invoice
2. Add Item with a Qty more than 1
3. Save Invoice as Draft
4. Edit Invoice and you will see it overides the Qty to 1, not the inputted Value.

Expected:
This is should keep the originally inputted amount.

**Comment by thicolares** [comment]
@pierredup can you assign this one to me? I was able to reproduce it.

**Comment by pierredup** [comment]
Thanks, @thicolares!

**Comment by thicolares** [comment]
I have faced two issues while installing Node dependencies after creating the container from the local Dockerfile.
This is how I have solved them so far:

### 1. admin-lte@3.0.5 install
```
npm install
...

// cat the xx.debug.log
8261 error code ELIFECYCLE
8262 error errno 1
8263 error admin-lte@3.0.5 install: `npm run plugins`
8263 error Exit status 1
8264 error Failed at the admin-lte@3.0.5 install script.
8264 error This is probably not a problem with npm. There is likely additional logging output above.
8265 verbose exit [ 1, true ]

```

Container:
```
root@471cdab71ea2:/opt/srv# npm -v
6.13.4
root@471cdab71ea2:/opt/srv# node -v
v8.17.0
````

Fixed this way:
```
root@471cdab71ea2:/opt/srv# nvm install 12.19.0
root@471cdab71ea2:/opt/srv# nvm use 12.19.0
```

I ended up updating npm too: `npm install npm@latest` (but the problem was already solved).

### 2. node-sass 
```
root@471cdab71ea2:/opt/srv# npm run dev
...
Error: ENOENT: no such file or directory, scandir '/opt/srv/node_modules/node-sass/vendor'
````

Fixed this way:
```
npm rebuild node-sass
```

**Comment by thicolares** [comment]
@pierredup it happens on `2.0.4`, but I cannot reproduce it on `2.1.0-dev`. I can say this bug was already fixed.
I noticed also the default value of Qtd is empty instead of `1`.

**Comment by pierredup** [comment]
This has been fixed in #347 and will be part of the upcoming 2.1 release.
